.gitignore

```diff
@@ -317,4 +317,10 @@ paket-files/
 __pycache__/
 *.pyc
 
+amalgamation_demo.c
+amalgamation_demo.cpp
+roaring.c
```

Comment:
Why do these three files ignored?

Reply:
They are automatically generated files that are not part of our source files. See the amalgamation script.

Reply:
Oh, the amalgamated file is $ROOT/roaring.c instead of src/roaring.c. The .gitignore file functions correctly in Git, but tools like fd and ripgrep somehow also ignore src/roaring.c. It will work if we explicitly specify them like this in .gitignore:

```
/amalgamation_demo.c
/amalgamation_demo.cpp
/roaring.c
/roaring.h
/roaring.hh
```

Reply:
That's a good point, it actually only works in gitignore because git only considers the gitignore when it would _add_ a file to be tracked, changes to existing files already in git are tracked regardless of what's in .gitignore.